includes/class-wc-ajax.php

```diff
@@ -378,7 +378,7 @@ public static function update_order_review() {
 			array(
 				'result'    => empty( $messages ) ? 'success' : 'failure',
 				'messages'  => $messages,
-				'reload'    => $reload_checkout ? 'true' : 'false',
+				'reload'    => $reload_checkout,
```

Comment:
This change seems unrelated to what this PR propose.

Reply:
@claudiosanches Yes, you're right. Sorry, I'm new to GitHub and don't know why this commit is showing in this PR. I've created this commit and opened another PR (#23891) to fix issue (#23888). Can you please guide me to resolve this? And let me know what possibly caused this, so in future I don't make same mistake. Thanks.

Reply:
Seems like you included an extra commit in here. You can use https://git-scm.com/docs/git-revert to revert it.

Reply:
I think I have resolved the issue regarding the extra commit from another branch. I guess, this happened because I forgot to change branch to master before creating new branch while I was still checked out on the other branch for another issue, so it brought commits from that branch too into this. I used `git rebase -i HEAD~3` and removed that extra commit and then force pushed to origin.
I think, now this PR is ready to be merged. :)